monorepo-trigger/README.md

```diff
@@ -0,0 +1,36 @@
+---
+title: CI for monorepo
+description: Conditionally trigger pipelines within a monorepo based on changes to file paths.
```

Comment:
Not 100% sold on what I've written here. Very happy for suggestions and improvements.

Reply:
for monorepo**s** ?

Reply:
I reckon plural might be more appropriate at the category level.

Reply:
oh right, this was description. Yeah OK `for a monorepo` is better.